plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt

```diff
@@ -65,8 +65,9 @@ class PointAnnotation(
     set(value) {
       value?.let {
         field = it
-        if (iconImage == null) {
-          iconImage = ICON_DEFAULT_NAME_PREFIX + it.generationId
+        if (iconImage == null || iconImage!!.startsWith(ICON_DEFAULT_NAME_PREFIX)) {
```

Comment:
Why was second condition added?

Reply:
If users set `iconImageBitmap` before, `iconImage` would be set to a name start with `ICON_DEFAULT_NAME_PREFIX`. And then users set a new `iconImageBitmap`, we need to update `iconImage` to the new bitmap, otherwise, it will still show the previous bitmap.

Reply:
I see